python/lsst/ts/ATWhiteLightSource/fakeChillerComponent.py

```diff
@@ -4,27 +4,29 @@
 
 
 class FakeChillerComponent(object):
-    def __init__(self, ip, port):
+    def __init__(self, ip, port, com_lock, log):
         self.ip = ip
         self.port = port
         self.timeout = 5
         self.con = False
-        self.response_dict = \
-            {b'.0103rSetTemp26\r': b'#01030rSetTemp+020038\r',
-             b'.0101WatchDog01\r': b'#01010WatchDog1001E8\r',
-             b'.0120rWarnLv1ee\r': b'#01200rWarnLv10800DB\r',
-             b'.0104rSupplyT46\r': b'#01040rSupplyT+028464\r',
-             b'.0107rReturnT3c\r': b'#01070rReturnT+012352\r',
-             b'.0108rAmbTemp0f\r': b'#01080rAmbTemp+031124\r',
-             b'.0109rProsFlo2f\r': b'#01090rProsFlo+001949\r',
-             b'.0110rTECB1Cr66\r': b'#01100rTECB1Cr+000177\r',
-             b'.0111rTECDrLvb7\r': b'#01110rTECDrLv+0000C7\r',
-             b'.0113rTECB2Cr6a\r': b'#01130rTECB2Cr000,C8E\r',
-             b'.0149rUpTime_21\r': b'#01490rUpTime_1362627A\r',
-             b'.0150rFanSpd1d3\r': b'#01500rFanSpd10000B8\r',
-             b'.0151rFanSpd2d5\r': b'#01510rFanSpd20000BA\r',
-             b'.0152rFanSpd3d7\r': b'#01520rFanSpd30000BC\r',
-             b'.0153rFanSpd4d9\r': b'#01530rFanSpd40000BE\r'}
+        self.log = log
+        self.response_dict = {
```

Comment:
Are you sure you want to keep the trailing `\r`? might be best to strip the string...

Reply:
I agree and also suggest making them ordinary strings. Encode them and add the \r when you use them.

Reply:
what I did was run these tests with the actual chiller hardware and then captured the output, including the return character. I left them in since thats what actual communication from the chiller looks like.

Reply:
well, yes, but the trailing `\r` is actually "carriage return", indicating "end of string". So the actual package you want is everything before that.

Reply:
This is another example where removing the terminator and decoding right away helps. You can work with plain strings and remove the "yes I know it was sent, but I don't want it" terminator.